docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -31,7 +31,8 @@ signs transaction with a key stored in an encrypted file or an external vault (f
 
 ### `chain-id`
 
-Chain ID of the network to receive the signed transactions.
+[Chain ID]()https://besu.hyperledger.org/en/stable/Concepts/NetworkID-And-ChainID/ of the network to
```

Comment:
Link is bit broken here, brackets aren't around the hyperlink

Reply:
Done